sky/cli.py

```diff
@@ -1607,10 +1607,33 @@ def logs(
 def cancel(cluster: str, all: bool, jobs: List[int]):  # pylint: disable=redefined-builtin
     # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
     """Cancel job(s)."""
+    bold = colorama.Style.BRIGHT
+    reset = colorama.Style.RESET_ALL
+    if not jobs and not all:
+        # Friendly message for usage like 'sky cancel 1'.
+        raise click.UsageError(
+            f'Use {bold}sky cancel <cluster> <job ID>{reset} '
+            f'or {bold}sky cancel <cluster> --all{reset} to cancel one '
+            'or all jobs on a cluster. Job IDs can be looked up by '
+            f'{bold}sky queue{reset}.')
```

Comment:
Previously, `sky cancel <cluster>` could be used to cancel the last job on the cluster. Should we only show the message when the `<cluster>` does not exist in the cluster table, or the `<cluster>` is an integer?

Reply:
Ahh, makes sense! That sounds good to me.